### ISSUE TITLE
Color contrast miss?

### ISSUE THREAD

**Issue by johnfoliot** [dequelabs/axe-core#600]
This one is weird, because Gokhan and I are getting "different" results from Assure.

Visit https://fm.info4.citi.com/ats/msg.aspx?sg1=40f21d9a1621103428227fe75a6c9a85

Down in the bottom left corner is some text (#cccccc on #ffffff) that did not get picked up by the tool

(hope this works: https://assure.dequecloud.com/test-run/24d579a4-bf4e-11e7-b8a8-e34fd6534073/pages/2/auto)
![image](https://user-images.githubusercontent.com/1228403/32384439-e4d80a7e-c088-11e7-985f-7b73f9130da2.png)

**Comment by johnfoliot** [comment]
Gokhan's mobile test:
https://assure.dequecloud.com/test-run/622ec8b4-c09d-11e7-8405-876fca4bc529/issue/692f1e20-c09d-11e7-a4ef-cb178f8ec699?filter%5Brule%5D=auto&filter%5Bpage_number%5D=0&sortField=ordinal&sortDir=asc&row=0

**Comment by marcysutton** [comment]
axe-core isn't missing it at a desktop viewport for me; I get that text as a color contrast violation. I don't know enough about what Assure does with the axe-core results–could it be in the incomplete array and not shown in the Assure UI for some reason? Incomplete can happen if there was an exception in execution of the rule. I didn't get that result in the aXe extension, but it's one possibility related to the JavaScript API.

We could use some more details about the conditions the page was tested under. Browser version and viewport size would help.

**Comment by marcysutton** [comment]
@johnfoliot can you provide more details? Otherwise I'll have to close this, as axe-core is reporting the expected result.
